Makefile.toml

```
@@ -43,3 +42,36 @@ dependencies = [
     "clippy",
     "docs"
 ]
+
+# testing
+[tasks.test]
+command = "cargo"
+args = ["test", "--all-features", "--workspace", "--", "--nocapture"]
+
+# docker
+[tasks.docker-build-node]
+workspace = false
+script = '''
+CREATED=$(date)
+VERSION=$(cat node/Cargo.toml | grep -m 1 '^version' | cut -d '"' -f 2)
+COMMIT=1
```

```suggestion
COMMIT=$(git rev-parse HEAD)
```

should do the trick for this one